plugins/vendor/zoom/infra/kZoomClient.php

```diff
@@ -112,7 +112,7 @@ public function deleteRecordingFile($meetingUUid, $recodingId)
 	public function listRecordings($accountId, $from, $to, $nextPageToken, $pageSize)
 	{
 		$apiPath = str_replace('@accountId@', $accountId, self::API_LIST_RECORDING);
-		$apiPath .= '?from=' . $from . '&to=' . $to . '&next_page_token=' . $nextPageToken . '&pageSize=' . $pageSize;
+		$apiPath .= '&page_size=' . $pageSize . '&next_page_token=' . $nextPageToken . '?from=' . $from . '&to=' . $to;
```

Comment:
In the api call of Zoom get recording ( https://marketplace.zoom.us/docs/api-reference/zoom-api/cloud-recording/getaccountcloudrecording ) the account id is part of the request parameters and the rest of parameters are query parameters, meaning that they must come after the - ?
This is how a correct call should look like '?from=' . $from . '&to=' . $to . '&next_page_token=' . $nextPageToken . '&page_size=' . $pageSize;